src/test/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsControllerTests.java

```diff
@@ -29,6 +29,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
```

Comment:
unused import?

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/99/commits/6bf9a89ee8ac77be504430c2e470ca00269ebc6d